api/src/test/java/uk/gov/dhsc/htbhf/hmrc/entity/HouseholdFactory.java

```diff
@@ -2,16 +2,18 @@
 
 import java.time.LocalDate;
 
+import static uk.gov.dhsc.htbhf.hmrc.testhelper.TestConstants.*;
+
 public class HouseholdFactory {
 
     public static Household aHousehold() {
         return aHouseholdWithNoAdultsOrChildren()
                 .build()
-                .addAdult(anAdult("Homer", "Simpson", "QQ123456C"))
-                .addAdult(anAdult("Marge", "Simpson", "QQ123456D"))
-                .addChild(aChild("Bart", "Simpson", 48))
-                .addChild(aChild("Lisa", "Simpson", 24))
-                .addChild(aChild("Maggie", "Simpson", 6));
+                .addAdult(anAdult(HOMER_FORENAME, SIMPSONS_SURNAME, HOMER_NINO))
+                .addAdult(anAdult(MARGE_FORENAME, SIMPSONS_SURNAME, MARGE_NINO))
+                .addChild(aChild("Bart", SIMPSONS_SURNAME, 48))
+                .addChild(aChild("Lisa", SIMPSONS_SURNAME, 24))
```

Comment:
You have a `LISA_FORENAME` constant you can use.

Reply:
Updated to use `SIMPSONS_SURNAME`